src/System.Reflection.Metadata/specs/PortablePdb-Metadata.md

```diff
@@ -499,5 +500,18 @@ The number of lambda entries is determined by the size of the blob (the reader s
 
 The exact algorithm used to calculate syntax offsets and the algorithm that maps lambdas/closures to their implementing methods, types and syntax nodes is language and implementation specific and may change in future versions of the compiler.
 
+##### <a name="EmbeddedSource"></a>Embedded Source (C# and VB compilers)
+Parent: Document
 
+Kind: {0E8A571B-6926-466E-B4AD-8AB04611F5FE}
 
+Embeds the text of the corresponding document in the PDB
+
+The blob has the following strucutre:
+
+    Blob ::= format content
+    
+| terminal  | encoding | description|
+|:----------|:---------|:-----------|
+| _format_  | uint16   | Indicates how the content is serialized. 0 = raw bytes, uncompressed. 1 = compressed in GZIP format. Other values reserved for future formats. |
+| _text_    | uint8[]  | The text of the document in the specified format. The length is implied by the length of the blob minus two bytes for the format.
```

Comment:
```content``` instead of ```text``` 
Nit: I'd specify "blob" or "format specific" for encoding instead of ```uint8[]```. It's kind of unclear what ```[]``` means within the context of the metadata spec.